### PR TITLE
Adding ASF License header to Python and Golang Drivers source files

### PR DESCRIPTION
**Pull request by rhizome-ai**
I added ASF License header to all python and golang drivers source files.

### CODE REVIEW

**Comment by rhizome-ai** [comment]
To match the other drivers, I changed ANTLR grammar for parser to Agtype.g4.

File: drivers/golang/README.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ Check [latest version](https://github.com/apache/incubator-age/releases)
 * Apache Incubator Age : https://age.apache.org/
 * Github : https://github.com/apache/incubator-age
 * Document : https://age.incubator.apache.org/docs/
+* apache-age-go GitHub : https://github.com/rhizome-ai/apache-age-go
```
Comment (JoshInnis) [inline drivers/golang/README.md:30]: URL needs to be updated

File: drivers/python/age/gen/AgtypeVisitor.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
+# Generated from Agtype.g4 by ANTLR 4.9.2
```
Comment (JoshInnis) [inline drivers/python/age/gen/AgtypeVisitor.py:1]: Needs License

File: drivers/python/age/gen/AgtypeListener.py [hunk 3]
```diff
@@ -0,0 +1,147 @@
+# Generated from Agtype.g4 by ANTLR 4.9.2
```
Comment (JoshInnis) [inline drivers/python/age/gen/AgtypeListener.py:1]: Needs License

File: drivers/python/age/gen/AgtypeVisitor.py [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+from antlr4 import *
```
Comment (JoshInnis) [inline drivers/python/age/gen/AgtypeVisitor.py:2]: Needs License

Comment (rhizome-ai) [reply drivers/python/age/gen/AgtypeVisitor.py:16]: I added license comments remained.

**Pull request merged**